autoinstrumentation/nodejs/src/autoinstrumentation.ts

```diff
@@ -0,0 +1,12 @@
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-otlp-grpc';
```

Comment:
From what I understand NodeJS doesn't support env vars like OTEL_TRACES_EXPORTER yet. When it does, we would remove this sort of hard-coding.

Reply:
Should this be an issue on this repository then?